### DISCUSSION TITLE
Make issue forms available in GitHub app in iOS/Android

### DISCUSSION THREAD

**Discussion by damian-krawczyk** [community/community#4793] [Projects and Issues]
Do you plan to make possible creation of issues with issues forms via GitHub app in iOS/Android? If so, what’s the ETA?

**Comment by lukehefson** [comment]
Hey @damian-krawczyk! Yes, we definitely intend to support issue forms in the Mobile apps at some point in the near future. We still have some work to do on the API implementation side for Mobile as well as figuring out the right UX – but it’s definitely something we want to do! At this point I wouldn’t be able to give you an ETA I’m afraid!

**Reply by Dusch4593** [reply]
It looks like we can't access existing issue templates on GitHub for iOS. We just merged a PR to use templates and I'm getting a blank screen when I tap "+" under "Issues". Here's a screenshot:

![IMG_D808E65ED466-1](https://user-images.githubusercontent.com/23144845/142428938-79c60bf3-64f2-4c77-a503-4f8c240b68f4.jpeg)

Fwiw

**Reply by Iron-Ham** [reply]
@Dusch4593 could you let me know what version of the iOS app you're using? That seems to be a newly reported bug for the iOS app. 

You can check your version by going to the bottom of the app's settings page via your user profile.

**Reply by Dusch4593** [reply]
@Iron-Ham EDIT: Oops, I think I misunderstood your question. Sorry about that. 😢 

I'm using mobile version 1.40.0 (52153418) of the GitHub app for iOS. 😄

**Reply by lukehefson** [reply]
> we can't access **existing** issue templates on GitHub for iOS

@Dusch4593 Looking at https://github.com/Codecademy/docs/tree/d646fdaf1adc8a247eacf9abf792e620fabe8a0b/.github/ISSUE_TEMPLATE it appears that Codecademy/docs _only_ has issue forms (not markdown issue templates – which currently **are** supported in Mobile) and blank issues turned off. So as issue forms are not yet supported in Mobile this unfortunately results in what you are seeing.

**Reply by Dusch4593** [reply]
@lukehefson Correct, blanks are turned off and we're only using yaml issue forms right now. Thanks for looking into this! Appreciated!

You probs already mentioned this on the thread but is this a feature that is coming soon? Probs when issue forms are out of beta?

**Comment by Wladefant** [comment]
Well at least github mobile now forces you to use github forms in the web browser. That is good and spares many hours for developers.

Bug: other clients with the api are able to create a blank issue without using forms.

**Comment by tom9358** [comment]
Are there any updates on this?
